pkg/kn/commands/broker/delivery_option_flags.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2022 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package broker
+
+import (
+	"github.com/spf13/cobra"
+	"knative.dev/client/pkg/kn/commands/flags"
+)
+
+type DeliveryOptionFlags struct {
+	SinkFlags     flags.SinkFlags
+	RetryCount    int32
+	Timeout       string
+	BackoffPolicy string
+	BackoffDelay  string
+	RetryAfterMax string
+}
+
+func (d *DeliveryOptionFlags) Add(cmd *cobra.Command) {
+	d.SinkFlags.AddWithFlagName(cmd, "dl-sink", "")
+	cmd.Flag("dl-sink").Usage = "Reference to a sink for delivering events that can not be sent"
```

Comment:
Well I'm not sure if shouldn't keep the default usage with all the explanation of sinks URIs.